src/main/java/fm/last/moji/impl/FileUploadOutputStream.java

```diff
@@ -90,12 +92,17 @@ public void close() throws IOException {
           delegate.close();
         } finally {
           try {
-            String message = httpConnection.getResponseMessage();
             int code = httpConnection.getResponseCode();
             if (HttpURLConnection.HTTP_OK != code && HttpURLConnection.HTTP_CREATED != code) {
-              throw new IOException(code + " " + message);
-            } else {
-              log.debug("Status: HTTP {} - {}", code, message);
+              InputStream is;
+              if (200 <= httpConnection.getResponseCode() && httpConnection.getResponseCode() <= 299) {
```

Comment:
You alread have the value for httpConnection.getResponseCode() in the variable "code" so re-use this rather than call the getter each time, code will be more concise. Please put the number on the right hand side in both comparisons so it's consistent and easier to read. Please replace the magic numbers of 200 and 299 with constants so that it's clear what they are? Finally is it safe to use greater/less than comparisons to capture all cases that don't result in an error stream or is there a more precise way to express this?